src/test/scala/it/bitrock/dvs/api/core/dispatcher/GlobalMessageDispatcherSpec.scala

```diff
@@ -59,7 +59,7 @@ class GlobalMessageDispatcherSpec extends BaseTestKit {
               )
             )
           )
-          messageProcessor ! CoordinatesBox(49.8, -3.7, 39.7, 23.6)
+          messageProcessor ! CoordinatesBox(49.8, -3.7, 39.7, 23.6, None)
```

Comment:
always None? not even one Some in a test?

Reply:
It's just a first part, where I am testing compatibility. I plan to add test when extra field is present.

Reply:
Added test for throttle